.gitignore

```diff
@@ -1 +1,2 @@
 codecept.phar
+node_modules
```

Comment:
While we are not installing Node modules at this time in `dev-lib` it is likely that we will in the future (for tests and such).